source/handshake.c

```diff
@@ -140,17 +196,27 @@ int main(int argc, char * const argv[])
     if (NUM_CALLS_PER_TEST % threadcount > 0) /* round up */
         num_calls += threadcount - NUM_CALLS_PER_TEST % threadcount;
 
-    if (share_ctx == 1) {
-        if (!perflib_create_ssl_ctx_pair(TLS_server_method(), TLS_client_method(),
-                                         0, 0, &sctx, &cctx, cert, privkey)) {
-            printf("Failed to create SSL_CTX pair\n");
+    if (per_thread_ossl_lib_ctx) {
+        int ret = perflib_run_multi_thread_test(do_handshake_ossl_lib_ctx_per_thread, threadcount, &duration);
+        if (!ret) {
```

Comment:
Nit: Add empty line after declarations and split long lines.

Reply:
Thanks, updated.

Reply:
You did not add the empty line after the declaration.

Reply:
I have updated the MR.